### PR TITLE
feat: Add unit tests for rapyd controllers and services

### LINKED ISSUE #687: Rapyd controller tests
Rapyd controller tests should be created.
Rapyd Service tests can be also added if needed.

### PR DESCRIPTION
**Pull request by rico191013**
### Context

- fixes #687

### CODE REVIEW

File: packages/wallet/backend/tests/mocks.ts [hunk 1]
```diff
@@ -35,6 +37,180 @@ export const mockSignUpRequest = (
...
+    createWallet: rapydFailResponse,
+    verifyIdentity: rapydFailResponse,
+    updateProfile: rapydFailResponse
+  }
+}
+
+export const mockedRapydService = {
+  getCountryNames: () => ({
+    lable: faker.location.country(),
+    valeu: faker.location.countryCode('alpha-2')
```
Comment (raducristianpopa) [inline packages/wallet/backend/tests/mocks.ts:187]:
```suggestion
    value: faker.location.countryCode('alpha-2')
```

File: packages/wallet/backend/tests/rapyd/controller.test.ts
Comment (raducristianpopa) [inline packages/wallet/backend/tests/rapyd/controller.test.ts:1]: We should also include some tests that simulate failures.

File: packages/wallet/backend/tests/rapyd/controller.test.ts [hunk 2]
```diff
@@ -0,0 +1,318 @@
...
+    it('should return a random country name', async () => {
+      await rapydController.getCountryNames(req, res, next)
+      expect(res.statusCode).toBe(200)
+      expect(res._getJSONData()).toMatchObject({
+        success: true,
+        message: 'SUCCESS'
+      })
+    })
+
+    it('should fail with status 500 on unexpected error', async () => {
```
Comment (dragosp1011) [inline packages/wallet/backend/tests/rapyd/controller.test.ts:136]: what is the unexpected error?

File: packages/wallet/backend/tests/rapyd/service.test.ts [hunk 3]
```diff
@@ -0,0 +1,235 @@
...
+    knex.destroy()
+  })
+
+  afterEach(async (): Promise<void> => {
+    await truncateTables(knex)
+    Reflect.set(rapydService, 'deps', mockRapyd)
+  })
+
+  describe('Get Document Types', () => {
+    it('Should return an array of RapydDocumentType', async () => {
```
Comment (dragosp1011) [inline packages/wallet/backend/tests/rapyd/service.test.ts:65]:
```suggestion
    it('should return an array of RapydDocumentType', async () => {
```

**Review by raducristianpopa — APPROVED** [review]

**Review by dragosp1011 — APPROVED** [review]

**Review by beniaminmunteanu — APPROVED** [review]

**Pull request merged**